config/config-network.yaml

```diff
@@ -173,12 +173,34 @@ data:
     # Knative doesn't know about that otherwise.
     default-external-scheme: "http"
 
+    # internal-encryption is deprecated and replaced by internal-dataplane-trust and internal-controlplane-trust
     # internal-encryption indicates whether internal traffic is encrypted or not.
-    # If this is "true", the following traffic are encrypted:
-    #  - ingress to activator
-    #  - ingress to queue-proxy
-    #  - activator to queue-proxy
     #
     # NOTE: This flag is in an alpha state and is mostly here to enable internal testing
     #       for now. Use with caution.
     internal-encryption: "false"
+
+    # internal-dataplane-trust indicates the level of trust established in the knative data-plane.
+    # internal-dataplane-trust = "disabled" (the default) - uses no encryption for internal traffic
+    # Using any other value ensures that the following traffic is encrypted using TLS:
+    #  - ingress to activator
+    #  - ingress to queue-proxy
+    #  - activator to queue-proxy
+    #
+    # internal-dataplane-trust = "minimal" ensures data messages are encrypted, Kingress authenticate that the receiver is a Kserv
```

Comment:
nit: Please change `Kserv` to `Ksvc` or `Kservice`. IMO, `Kserv` or `Kserve` is not used in general and it might be confused with  https://github.com/kserve/kserve